README.rst

```diff
@@ -18,6 +18,27 @@ The recommended way to install this plugin is to use ``pip``::
 Note that this plugin requires Python 3.7+ and Sopel 7.1+. It won't work on
 Python versions that are not supported by the version of Sopel you are using.
 
+Configure
+=========
+
+``sopel-iplookup`` can be configured by invoking Sopel's interactive wizard:
+
+    $ sopel-plugins configure iplookup
+    Configure Sopel GeoIP Lookup Plugin
+    Please consult sopel-iplookup's README to learn about its settings.
+
+    Path to existing GeoIP db files (leave empty to auto download): 
+    MaxMind license key (optional): (hidden input)
```

Comment:
I think that's meant to be a code block?

````suggestion
```
$ sopel-plugins configure iplookup
Configure Sopel GeoIP Lookup Plugin
Please consult sopel-iplookup's README to learn about its settings.

Path to existing GeoIP db files (leave empty to auto download): 
MaxMind license key (optional): (hidden input)
```
````

Reply:
This is rST, not Markdown.

I'm not against changing the readme to Markdown, though. It'd be simpler to test-render with `grip` or a VS Code extension. (Couldn't get the one I found for rST to work.)

Reply:
Sigh, right, my classic mistake 🤦

It does render incorrectly, though, I guess it needs an extra colon in the preceding paragraph

![image](https://github.com/sopel-irc/sopel-iplookup/assets/6125322/fc5d49d3-7fb1-43ba-ab65-37c61af2c6f7)

Reply:
Oh, so it does. Without the double colon it's taken as a blockquote, I guess.

Reply:
Should be fixed in the last push.